aws/iam/helpers.py

```diff
@@ -0,0 +1,85 @@
+from dateutil.parser import parse
+
+
+def user_is_inactive(iam_user, grace_period, considered_inactive):
+    """
+    Returns False if any of these are true:
+        - The user was created less than the grace period
+        - The user has used either possibly active access keys since the date
+          that is "considered_inactive"
+        - The user has logged into the AWS console since the date that is
+          "considered_inactive"
+    else it will return True.
+
+    >>> from datetime import datetime
+    >>> grace_period = datetime(2018, 1, 8)
```

Comment:
Assuming the above comment about whether I'm understanding `grace_period` correctly then I'd expect `grace_period` to be a time interval timedelta and we'd compare something like: `parse(iam_user['user_creation_time']) + grace_period >= datetime.now()` on L71.

OK, I see this is happening the config parsing, so maybe just renaming these and `considered_inactive` to be clearer like `created_after` and `no_activity_since` would work.

Reply:
Thank you, I was struggling a bit with how to name these two vars.